src/3d/qgs3dmapcanvas.cpp

```diff
@@ -13,6 +13,9 @@
  *                                                                         *
  ***************************************************************************/
 
+#include "qgsframegraph.h"
+#include "qgspointcloudlayer3drenderer.h"
+#include "qgsrubberband3d.h"
```

Comment:
can we have these includes together with other qgis includes a bit further below?